source/_patterns/molecules/interactive/accordion/accordion.twig

```diff
@@ -0,0 +1,24 @@
+{% set tag = tag|default('h2') %}
```

Comment:
When there are headings it is a good idea to use a variable for the heading level, so we can pass the appropriate level when including the template.